segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -68,11 +69,35 @@ public void close() throws Exception {
     //region BookKeeper operations
 
     /**
+     * Suspends the BookieService with the given index (does not stop it).
```

Comment:
Yes, it simulates the Bookie not being available by turning off its Netty interface.